doc/workflow/qpm_packages_installation.md

```diff
@@ -0,0 +1,11 @@
+To work with PPS 2.0 you need to install Font Awesome web fonts package manager, so:
+-  download [qpm package manager](https://www.qpm.io/). 
+-  put qpm.exe in PPS project folder.
+-  use `qpm install com.cutehacks.fontawesome` in command line.
+-  `include($$PWD/vendor/vendor.pri)` in pllug-presentation-system.pro file.
```

Comment:
You need to information, started from there, as we already added this code to our project.